libsigner/Cargo.toml

```diff
@@ -33,6 +33,7 @@ stackslib = { path = "../stackslib"}
 thiserror = "1.0"
 tiny_http = "0.12"
 wsts = { workspace = true }
+mutants = "0.0.3"
```

Comment:
Why is this now a dependency?  We shouldn't be linking it into the production binary.

Reply:
moved it to dev-dependency